src/instr/instr_paje_types.cpp

```diff
@@ -45,33 +45,14 @@ static type_t newType (const char *typeNameBuff, const char *key, const char *co
   return ret;
 }
 
-void PJ_type_free (type_t type)
+s_type::~s_type()
 {
-  value* val;
-  char *value_name;
-  xbt_dict_cursor_t cursor = nullptr;
-  xbt_dict_foreach (type->values, cursor, value_name, val) {
```

Comment:
How can I use delete instead of xbt_free, while the allocation is done with calloc?